client/src/components/ContentNavigator/index.ts

```diff
@@ -247,6 +256,32 @@ class ContentNavigator implements SubscriptionProvider {
           "workbench.actions.treeView.contentdataprovider.collapseAll",
         );
       }),
+      commands.registerCommand(
+        "SAS.convertNotebookToFlow",
+        async (resource: ContentItem) => {
+          // Open window to chose the name and location of the new .flw file
+          const name = await window.showInputBox({
+            prompt: "Enter the name for the new .flw file",
```

Comment:
This should have an entry in `Messages` as well

Reply:
done in 33035265761fa800663eeb3c9a709e589a66d92a